tests/src/test/java/alluxio/master/file/ConcurrentFileSystemMasterUtils.java

```diff
@@ -121,7 +122,10 @@ public void run() {
                 fileSystem.delete(paths[iteration]);
                 break;
               case GET_FILE_INFO:
-                fileSystem.getStatus(paths[iteration]);
+                URIStatus status = fileSystem.getStatus(paths[iteration]);
+                if (!status.isFolder()) {
+                  Assert.assertNotEquals(0, status.getBlockIds().size());
```

Comment:
This fails without your fix?

Reply:
Yes, though not deterministicly

Reply:
Would it be possible to add a targeted unit tests for the condition?

Reply:
The operation that triggers the issue is the concurrent metadata load of the single UFS file, which we actually have a test.